assets/scss/7-layout/_content-grid.scss

```diff
@@ -0,0 +1,32 @@
+// Content Grid (l-content-grid)
+//
+// Standard two col grid with a content area and sidebar. We use CSS variables to allow for flexibility in setting column ratios, order, and gap size. This grid starts at `bp-l`. The default sidebar width is to account for 300x250 ads.
+//
+// Markup: 7-layout/content-grid.html
+//
+// Styleguide 7.0.1
+//
+:root {
+  --content-grid-order: 'content side';
+  --content-grid-template: 1fr 310px;
+  --content-grid-gap: #{$size-xxl};
+}
+
+.l-content-grid {
+  @supports (display: grid) {
+    @include mq($from: bp-l) {
+      @include gap(var(--content-grid-gap));
+      display: grid;
+      grid-template-areas: var(--content-grid-order);
```

Comment:
If we did want to override these variables, where would be the proper place to do that? In a page-specific stylesheet?

Reply:
Good question. I don't know if I have a strong opinion on that. Repo or page-specific would be just fine. You could stick a variation class or beat the specificity with a parent class. I think they even work inline? I know introducing CSS variables without a set standard might be unwise, but I haven't used them enough to really know how they change our greater organization. They feel useful though right?

Reply:
I'm sure we'll figure it out as we go.